src/cpp_wrapper/Dictionary.h

```diff
@@ -24,7 +24,7 @@ ion_dictionary_handler_t	handler;
 ion_dictionary_t			dict;
 int							size_k;
 int							size_v;
-int							dict_size;
+unsigned int				dict_size;
```

Comment:
Ideally, these would all use `ion_dictionary_size_t` - how feasible would that be to do?